lonestar/analytics/cpu/boruvka/Boruvka.cpp

```diff
@@ -423,7 +422,10 @@ void run() {
 
 int main(int argc, char** argv) {
   galois::SharedMemSys G;
-  LonestarStart(argc, argv, name, desc, url);
+  LonestarStart(argc, argv, name, desc, url, inputFile.c_str());
+
+  galois::StatTimer totalTime("TimerTotal");
+  totalTime.start();
```

Comment:
Missing stop.

Since I'm seeing this somewhat consistently, I have to ask: is this being done on purpose? i.e. you're just letting the destructor take care of things. Does the destructor save the time when it gets destroyed, though?